### DISCUSSION TITLE
wxFormBuilder 3.5

### DISCUSSION THREAD

**Discussion by DBizners** [wxFormBuilder/wxFormBuilder#728] [Q&A]
Hi,

We are using wxFormBuilder 3.5 under Python 2.7.  We have 2 questions:

-may we upgrade wxFormBuilder to the latest version available (compat with 2.7) ? 

- where can we find the graphical GUI changes between the different versions ?

Thanks,

Bernard

**Comment by sodevel** [comment]
That is a very old wxFB version, one from before the move to Github, not sure which wxWidgets version that did target, at most 2.8, maybe even older? I also know nothing about Python, well, i do know that 2.7 is also very old and deprecated and there have been big breaking changes in Python 3.x.

The main question is not the compatibility with Python 2.7 but the wxWidgets language binding for Python, wxPython. In general, recent wxFB versions aim to support latest wxWidgets 3.1 API. I don't know how wxWidgets versions relate to wxPython, wxWidgets is currently at version 3.1.5 while wxPython is at version 4.1.1. The next question is, does wxPython still work on Python 2.7? Which is the latest version that does? And what wxWidgets API does that one support? These are questions i can't answer.

About a changelog, im afraid there is none beside the commit log.

**Comment by DBizners** [comment]
Hi Steffen,

Thank you very much for your feedback.  For your information, our wxPython
version is:  3.0.2.0 msw (classic)

From this post:
https://wxpython.org/pages/changes/index.html#:~:text=x)%20for%20the%20wxWidgets%20source,into%20keeping%20it%20that%20way
,
I understand we could use up to wxPython 4.1 to have binaries compatible
with Python 2.7.

In that case, what wxFB version could we use with wxPython 4.1 ?

Regards,

Bernard
